kong.conf.default

```diff
@@ -1213,12 +1213,35 @@
 # See the lua-nginx-module documentation for more information:
 # https://github.com/openresty/lua-nginx-module
 
-#lua_ssl_trusted_certificate =   # Absolute path to the certificate
-                                 # authority file for Lua cosockets in PEM
-                                 # format. This certificate will be the one
-                                 # used for verifying Kong's database
-                                 # connections, when `pg_ssl_verify` or
-                                 # `cassandra_ssl_verify` are enabled.
+
+#lua_ssl_trusted_certificate =   # Comma-separated list of paths to certificate
+                                 # authority files for Lua cosockets in PEM format.
+                                 #
+                                 # These certificates will be the combined
+                                 # into a single file during the startup phase.
+                                 #
+                                 # The special value `system` expands to the
+                                 # "usual default" provided by each distro, according
```

Comment:
```suggestion
                                 # The special value `system` attempts to search for the
                                 # "usual default" provided by each distro, according
```

seems to be better. "expands to the "usual default" provided by each distro" make it sounds like we insert all of these paths into the config, which may cause users to try to do the same by writing:

`lua_ssl_trusted_certificate = /path/to/a.pem, /path/to/b.pem` and expects us to ignore non existent path (which is not the behavior here).